src/state/trajectory/logics.ts

```diff
@@ -62,13 +62,7 @@ const resetSimulariumFileState = createLogic({
             if (controller) {
                 controller.clearFile();
             }
-            clearTrajectory = receiveTrajectory({
-                plotData: initialState.plotData,
-                firstFrameTime: initialState.firstFrameTime,
-                lastFrameTime: initialState.lastFrameTime,
-                timeStep: initialState.timeStep,
-                agentUiNames: initialState.agentUiNames,
-            });
+            clearTrajectory = receiveTrajectory(initialState);
```

Comment:
safe to pass by ref?

Reply:
This does seem like it could cause problems since when we change state we're setting individual keys, instead
```suggestion
            clearTrajectory = receiveTrajectory({...initialState});
```

Reply:
Yeah I wasn't sure about this, thanks for the input!